### PR TITLE
TEST: Add enhance-and-skullstrip regression tests

### PR DESCRIPTION
**Pull request by effigies**
Just to track progress on this, I think this is the rough framework we need for testing.

I picked this symmetric overlap metric (taken from [doi:10.1371/journal.pone.0007200](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0007200#article1.body1.sec4.sec3.p3)) because it's sensitive to increases and decreases in mask size, as well as shifts.

Next I'll need a list of images to make reference files from.

### CODE REVIEW

**Comment by effigies** [comment]
@oesteban Do you have a list of images? Otherwise I can randomly sample OpenFMRI on datalad.

**Comment by oesteban** [comment]
Oh, sorry. I think we can start with:

ds000237/sub-0[3,6] (#963)
ds000116/sub-12_task-visualoddballwithbuttonresponsetotargetstimuli_run-02 (#907)
ds000133/sub-06_ses-post_task-rest_run-01 (#905)
ds000140/sub-32_task-heatpainwithregulationandratings_run-02 (#904)
ds000157/sub-23_task-passiveimageviewing (#902)

I would also add the three echos from any of the participants in ME datasets (ds000210, ds000216).

And there was ds000148 with very short TEs which could be hard (#997).

You would probably need some reference that works. I'd take them from ds000030 which is a dataset we heavily massaged and generally works very well.

**Comment by chrisfilo** [comment]
Could you also add the case from #1181?

File: fmriprep/workflows/bold/tests/test_util.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+def test_masking(input_fname, expected_fname):
+    bold_reference_wf = init_bold_reference_wf(omp_nthreads=1, enhance_t2=True)
+    bold_reference_wf.inputs.inputnode.bold_file = input_fname
+    res = bold_reference_wf.run(plugin='MultiProc')
+
+    combine_masks = [node for node in res.nodes if node.name.endswith('combine_masks')][0]
+    overlap = symmetric_overlap(expected_fname,
+                                combine_masks.result.outputs.out_file)
+
+    assert overlap > 0.95, input_fname
```
Comment (effigies) [inline fmriprep/workflows/bold/tests/test_util.py]: My main remaining question is whether 0.95 is a good threshold. A 95% symmetric overlap means 95% of voxels overlap if the size stays the same; for an inflation where the old mask is a subset of the new, the inflation could be up to 110.8% (1/.95^2) the original size; or a deflation to as low as 90.25% (.95^2) of the original size.

Comment (oesteban) [reply fmriprep/workflows/bold/tests/test_util.py]: Seems like a good heuristic to start with :+1:

**Comment by effigies** [comment]
Okay, I'll go ahead and merge. We can refine the threshold along with getting #1050 to a mergeable state.

**Pull request merged**